indico/queries/export.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         frozen_labelset_ids: List[int] = None,
         combine_labels: LabelResolutionStrategy = LabelResolutionStrategy.ALL.name,
         file_info: bool = None,
-        anonymoous: bool = None,
+        anonymous: bool = None,
```

Comment:
lmao - I'm hoping no one used this otherwise technically a backwards-incompat change. will let @goatrocks make the final call here

Reply:
If we call it out clearly in the release notes, given this isn't something like the submission api... should be ok to fix 🐄  😨

Reply:
Found this library that support argument deprecation: https://docs.twisted.org/en/stable/api/twisted.python.deprecate.html

Don't know how worth it would be to add it for this

Reply:
or just print a `warnings.warn` if someone uses the old arg?

Reply:
warnings.warn is good -- we have a deprecation library already but unfortunately it doesn't support params (or it didn't when it was added). :(

Reply:
added typo argument back and checks for it as discussed

Reply:
Thank you Nate!